aws-connect-userhierarchygroup/src/main/java/software/amazon/connect/userhierarchygroup/BaseHandlerStd.java

```diff
@@ -58,7 +59,7 @@ protected abstract ProgressEvent<ResourceModel, CallbackContext> handleRequest(
     protected static void handleCommonExceptions(final Exception ex, final Logger logger) {
         if (ex instanceof ResourceNotFoundException) {
             throw new CfnNotFoundException(ex);
-        } else if (ex instanceof InvalidParameterException || ex instanceof InvalidRequestException) {
+        } else if (ex instanceof InvalidParameterException || ex instanceof InvalidRequestException || ex instanceof ResourceInUseException) {
```

Comment:
Do we need to add a unit test for this exception? I see that there are others for common exceptions in BaseHandlerStdTest.java.